addons/binding/org.openhab.binding.lutron/ESH-INF/thing/thing-types.xml

```diff
@@ -167,6 +167,154 @@
         </config-description>
     </thing-type>
 
+    <bridge-type id="prgbridge">
+        <label>Lutron GRX-PRG or GRX-CI-PRG Bridge</label>
+        <description>Ethernet access point to Lutron Grafik Eye 3x/4x Systems</description>
+
+        <channels>
+            <channel id="buttonpress" typeId="buttonpress"/>
```

Comment:
This is the two character code that specifies what button was pressed?  Would it be feasible to expose each button as its own channel and have the binding decode the two character code and trigger a button press event on the appropriate channel?